src/js/modules/api/showData.js

```diff
@@ -1,90 +1,79 @@
-import Popup from "../popup.js";
-import getLikes from "./getLikes.js";
-import postLikes from "./postLikes.js";
+/* eslint-disable no-unused-vars */
+import Popup from '../popup.js';
+import getLikes from './getLikes.js';
+import postLikes from './postLikes.js';
 
-const BaseUrl = "https://api.tvmaze.com/shows";
+const BaseUrl = 'https://api.tvmaze.com/shows';
 
 const ShowDetails = async () => {
-  const containersShows = document.querySelector(".container-movies");
+  const containersShows = document.querySelector('.container-movies');
```

Comment:
- [ ] [OPTIONAL] In this `showDetails` function you are doing a lot, try to split it up into smaller more comprehensible functions, and especially make sure the API call is in a separate function.